compiler/compile-config/compile-config-2.02-learning_inference

```diff
@@ -197,10 +197,11 @@ if .deepdive_.execution.processes | has("process/grounding/combine_factorgraph")
                 cd \"$DEEPDIVE_RESULTS_DIR\"/$pid
                 cat inference_result.out.text |
                 # restoring shard ID to the vids
-                jq -R -r --argjson SHARD_BASE $(($pid << 48)) '
-                    split(\" \") |
-                    [ (.[0] | tonumber + $SHARD_BASE | tostring)
-                    , .[1:][] ] | join(\"\\t\")' |
+                python3 -c "
```

Comment:
I think you need to escape the double quote (with backslash). Travis is complaining about jq compilation.

Also, @netj should we just use `python` here -- which should be compatible with both 2 and 3.

@zifeishan let's do some perf test for this?